src/main/resources/assets/minecolonies/lang/da_DK.lang

```diff
@@ -76,57 +68,78 @@ tile.minecolonies.blockHutFisherman.name=Fiskerhus
 tile.minecolonies.blockHutStonemason.name=Stenbrud
 tile.minecolonies.blockHutTownHall.name=Rådhus
-tile.minecolonies.substitutionBlock.name= SchematicHolder
-
+tile.minecolonies.blockSubstitution.name=
 tile.blockHut.messageNoPermission=Du har ikke nogen tilladelse til at åbne huse i %s.
-
+tile.blockHut.messageNoPermissionPlace=Du har ikke tilladelse til at placere en bygning %s
 com.minecolonies.job.Builder=Håndværker
 com.minecolonies.job.Deliveryman=Fragtmand
 com.minecolonies.job.Farmer=Landmand
 com.minecolonies.job.Lumberjack=Skovhugger
 com.minecolonies.job.Miner=Miner
 com.minecolonies.job.Fisherman=Fisker
-
 tile.blockHutTownHall.messageMaxSize=Max Indbygger Antal Nået
 tile.blockHutTownHall.messageColonistDead=En Indbygger Har Stillet Træskoene!
 tile.blockHutTownHall.messagePlacedAlready=Du Har Allerede Placeret Et Rådhus
 tile.blockHutTownHall.messageInvalidWorld=Du kan ikke placere at rådhus i denne type verden
 tile.blockHutTownHall.messageTooClose=For tæt på rådhuset (gå længere væk fra rådhuset)
-
+tile.blockHutTownHall.messageTooFar=Du er alt for langt væk fra din koloni
 tile.blockHut.messageTooFarFromTownHall=Du skal være tættere på rådhuset
 tile.blockHut.messageNoTownHall=Du skal først placere et rådhus
-
 item.minecolonies.supplyChestDeployer.name=Forsygningsskib's kiste
 item.minecolonies.scepterGold.name=Akitekt Værktøj
 item.minecolonies.scepterSteel.name=Scannings Værktøj
 item.minecolonies.caliper.name=Caliper
-
 item.supplyChestDeployer.invalid=Du skal være i nærheden af et større vandområde for at kunne placere kisten
-
 item.scepterSteel.point=Punkt Et Gemt
 item.scepterSteel.point2=Punkt To Gemt, Højre Click Igen For At Gemme Konstruktionen
 item.scepterSteel.samePoint=Det Var Det Samme Punkt
 item.scepterSteel.scanFormat=SCAN_%s-%s.Konstruktion
 item.scepterSteel.scanSuccess=Succes Konstruktionen Er Gemt Som %s
 item.scepterSteel.scanFailure=Scanningen Kunne IKKE Gemmes
-
 item.caliper.message.same=Det Er Den Samme Block.
 item.caliper.message.line=Det er en %d block linje.
 item.caliper.message.square=Det er en %d by %d Firkant.
 item.caliper.message.cube=Det er en %d by %d by %d Kasse.
-
 itemGroup.minecolonies=MineKoloni
-
 entity.worker.inventoryFullChestFull=der er ikke mere plads i min kiste. Vil du gerne lave plads i kisten, så jeg kan komme af med mine ting?
-
 entity.builder.messageBuildStart=Begyndt at Bygge %s
 entity.builder.messageBuildComplete=Blev Færdig Med At Bygge %s
 entity.builder.messageNeedMaterial=Jeg mangler %s (Amount: %d)
-
+entity.builder.messageBuilderNecessary=
 entity.miner.messageNeedTools=Mangler Værktøj %s!
 entity.miner.messageNeedBlockAndItem=Jeg mangler %s!
 entity.miner.messageNeedBetterTools=Jeg mangler bedre værktøj!
 entity.miner.messageMoreBlocks=Jeg skal ha flere %s!
 entity.miner.messageRequiresBetterHut=Upgradere mit hus så jeg kan komme længere ned!
-
 entity.fisherman.messageWaterTooFar=Fiskerhuset er for langt væk fra vandet!
+com.minecolonies.workOrderAdded=
```

Comment:
That would be great!

2016-08-29 9:50 GMT-03:00 Colton Schlosser <notifications@github.com>:

> In src/main/resources/assets/minecolonies/lang/da_DK.lang
> <https://github.com/Minecolonies/minecolonies/pull/68#discussion_r76600829>
> :
>
> >  entity.fisherman.messageWaterTooFar=Fiskerhuset er for langt væk fra vandet!
> > +com.minecolonies.workOrderAdded=
>
> Also do we want to comment out empty lines? So the english is used instead
> of blank string.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Minecolonies/minecolonies/pull/68/files/36b1b4b6ae76d4726b0154887313f9df8a5cc79e#r76600829>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGI9y03DA0qHcDezLSd5Ly8WEfhsyP3Fks5qktWegaJpZM4Ju8K->
> .
>